backstage/src/executor/events.ts

```diff
@@ -67,7 +71,7 @@ export function triggerEvent(gameContext: GameContext, event: MaybeInlineEvent,
       }
 
       console.debug(`[D] [triggerEvent] triggered event '${eventId}'`)
-      const hooks = gameContext.state.events.eventsTriggered[eventId]
+      const hooks = gameContext.state.events.eventTriggered[eventId]
```

Comment:
Suddenly I find this could make some dead loop. Fix it or leave it there and mark with a `TODO`.